test/common/concurrent_associative_common.h

```diff
@@ -425,6 +425,26 @@ void test_comparison_operators() {
     check_unequal(cont, cont6);
 }
 
+template <typename Range, typename Container>
+void test_empty_container_range(Container&& cont) {
+    REQUIRE(cont.empty());
+    Range r = cont.range();
+    REQUIRE_MESSAGE(r.empty(), "Empty container range should be empty");
+    REQUIRE_MESSAGE(!r.is_divisible(), "Empty container range should not be divisible");
+    REQUIRE_MESSAGE(r.begin() == r.end(), "Incorrect iterators on empty range");
+    REQUIRE_MESSAGE(r.begin() == cont.begin(), "Incorrect iterators on empty range");
+
+    Range r2(r, tbb::split{});
```

Comment:
It is not stated if split ctor can be used for non-divisible range; however, [the spec](https://spec.oneapi.io/versions/latest/elements/oneTBB/source/named_requirements/algorithms/range.html) says about `is_divisible`: "_True if the range can be partitioned into two subranges._". I.e. range cannot be partitioned if `false`. Does it mean that we cannot use splitting ctor to partition the empty range?